client/fixtures/cancelledProductDetail.ts

```diff
@@ -0,0 +1,29 @@
+import type { CancelledProductDetail } from '../../shared/productResponse';
+
+export const cancelledContribution: CancelledProductDetail = {
+	mmaCategory: 'recurringSupport',
+	tier: 'Contributor',
+	joinDate: '2022-01-05',
+	subscription: {
+		subscriptionId: 'A-S00303370',
+		cancellationEffectiveDate: '2023-03-20',
+		start: '2021-12-10',
+		end: '2022-11-29',
+		readerType: 'Direct',
+		accountId: '8ad09f8a7e25bda3017e296317464818',
```

Comment:
Just double checking these are mock/test account ids?

Reply:
I've updated the `subscriptionId` and `accountId` fields so that they're obviously mocked data to avoid any confusion in the future. I'll raise a separate PR to do the same for the existing fixtures. (Those will probably be slightly more work due to tests relying on specific account IDs being exposed in the UI.)